push.sh

```diff
@@ -1,7 +1,21 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-for package in $(find -name "*.nupkg" | grep "test" -v); do
-  echo "${0##*/}": Pushing $package...
-  dotnet nuget push $package --source https://f.feedz.io/carter/carter/nuget/index.json --api-key $1
+TARGET_PACKAGE=$1
+if [ "$TARGET_PACKAGE" = "carter" ]; then
+  TARGET_PACKAGE_PATH="./src/Carter/**/*.nupkg"
+elif [ "$TARGET_PACKAGE" = "newtonsoft" ]; then
+  TARGET_PACKAGE_PATH="./src/Carter.ResponseNegotiators.Newtonsoft/**/*.nupkg"
+fi
```

Comment:
Not a Bash expert by any means.

This is the way I found to target either the main `Carter` or the Newtonsoft package,